File: app/src/main/java/com/pkfcooperparry/template/data/model/api/BaseResponse.java
package com.pkfcooperparry.template.data.model.api;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

import lombok.Getter;
import lombok.Setter;

public class BaseResponse {

    @Getter
    @Setter
    @Expose
    @SerializedName("Error")
    private String message;


    public BaseResponse() {
    }


}

File: app/src/main/java/com/pkfcooperparry/template/data/model/api/BaseResponse.kt
package com.pkfcooperparry.template.data.model.api

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

import lombok.Getter
import lombok.Setter


open class BaseResponse {


    @Expose
    @SerializedName("Error")
    var message: String? = null


}
